src/main/java/Duke.java

```diff
@@ -1,10 +1,12 @@
+import ManagerClasses.InputManager;
+import ManagerClasses.PrintManager;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        PrintManager.showLogo();
+        PrintManager.showGreetMessage();
```

Comment:
Good use of [SLAP](https://nus-cs2113-ay2021s2.github.io/website/se-book-adapted/chapters/codeQuality.html#slap-hard) here.